evm/bridge-token-factory/contracts/BridgeTypes.sol

```diff
@@ -20,15 +20,15 @@ library BridgeTypes {
     }
 
     struct ClaimFeePayload {
```

Comment:
Can be removed?

Reply:
Yep, in next [PR](https://github.com/Near-One/omni-bridge/pull/117)